llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -16795,6 +16795,16 @@ bool AArch64TargetLowering::optimizeExtendOrTruncateConversion(
 
       DstTy = TruncDstType;
     }
+
+    // mul(zext(i8), sext) can be transformed into smull(zext, sext) when
+    // destination type is at least i32, which is faster than using tbl
```

Comment:
i32 is not mentioned below, would be good to update the comment to refer to the `SrcWidth * 4` below?